serotiny/datamodules/dataframe/utils.py

```diff
@@ -22,9 +22,9 @@ def get_canonical_split_name(split):
     raise ValueError
 
 
-def get_dataset(dataframe, transform, cache_dir=None):
+def get_dataset(dataframe, transform, split, cache_dir=None):
     data = list(dataframe.to_dict("records"))
-    if cache_dir is not None:
+    if cache_dir is not None and split in ["train", "val"]:
```

Comment:
sorry for the nitpick but can we put these in a tuple instead of a list?

Reply:
sure!